contributions/catalog/webapps/templates/contribute/contribute.html

```diff
@@ -788,119 +1101,262 @@ <h3 id="capability">Add a Capability to this Contribution</h3>
-                                {% else %}
+                        {% if is_editable %}
+                            {% for talent in post["talents"] %}
+                                <div class="form-group">
+                                    <label class="required control-label col-sm-2">Talent Name</label>
+                                    <div class="col-sm-10">
+                                        <input class="form-control" id="talent_name" name="talent_name"
+                                               placeholder="Give a name for your talent"
+                                               type="text" value="{{ talent|filter_nested_dict(["name"]) }}">
+                                    </div>
+                                </div>
+                                <div class="form-group">
+                                    <label class="required control-label col-sm-2">Talent Description</label>
+                                    <div class="col-sm-10">
+                                        <input class="form-control" id="talent_description" name="talent_shortDescription"
+                                               placeholder=""
+                                               type="text" value="{{ talent|filter_nested_dict(["shortDescription"]) }}">
+                                    </div>
+                                </div>
+                                <div class="form-group">
+                                    <label class="control-label col-sm-2">Required Building Block(s)</label>
+                                    <div class="col-sm-10">
+                                        {# TODO what is required building blocks #}
+                                        {% if talent["requiredBuildingBlocks"]|length > 0 %}
+                                        <input class="form-control" id="required_bb" name="talent_requiredBuildingBlocks"
+                                               placeholder=""
+                                               type="text" value="{{ talent|filter_nested_dict(["requiredBuildingBlocks"]) }}">
+                                        {% else %}
+                                            <input class="form-control" id="required_bb" name="talent_requiredBuildingBlocks"
+                                                   placeholder=""
+                                                   type="text" value="">
+                                        {% endif %}
+                                    </div>
+                                </div>
+                                <div class="form-group">
+                                    <label class="control-label col-sm-2">Required Capabilit(y/ies)</label>
+                                    <div class="col-sm-10">
+                                        <select class="custom-select" multiple id="required_capability" name="talent_requiredCapabilities">
```

Comment:
There is a submission error when posting a Contribution containing a Talent.  
This is the error in Contributions Catalog logs:
```
2021-11-17T21:06:06.209Z ERROR   [Thread-7  ] : root - Contribution POST {"detail": "0 is not of type 'object' - 'talents.0.requiredCapabilities.0'", "status": 400, "title": "Bad Request", "type": "about:blank"}
2021-11-17T21:06:06.209Z ERROR   [Thread-7  ] : root - post method fails with error: 400: ('400', 'Bad Request', "0 is not of type 'object' - 'talents.0.requiredCapabilities.0'")
```
This is the error in the Contributions BB logs:
``` 
validation error: 0 is not of type 'object' - 'talents.0.requiredCapabilities.0'
```
I want to clarify that `requiredCapabilities` is not a required field and hence it can contain an empty list.

A Contribution containing a Capability can be submitted just fine.

Please let me know if you want to create a new issue and work on it after your current tasks related to auth middleware library. We can merge this PR in that case. Thanks.

Reply:
I checked it, and it was because the required capabilities has been changed to have the link instead of the required capability itself. Let's merge this and work this on later. I believe that I handled it already, maybe in other PR. Or maybe I am wrong. Anyway, we can work on it later I guess.

Reply:
Thanks. I have created an issue here: #827.